lib/promises.helper.js

```diff
@@ -1,55 +1,98 @@
 "use strict";
 
-var prototype;
-
 /**
- * Wrap what is passed into one of the global functions in order to handle
- * promises.
+ * Allows .then to be chained onto method when testing promises, rather than
```

Comment:
This description is inaccurate.  The function itself wraps obj[methodName] and handles when a Promise object is returned.  If one is returned, it will automatically turn the test into an asynchronous test that finishes when the Promise is resolved.  When the promise is rejected the test will be failed.